topic/Azure function - event hub triggered/Azure Function/value_cache.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+class ValueCache:
+    """
+    ValueCache class is used to structure enriched data for insert.
+    """
+
+    def __init__(self):
+        self.errors = []
+        self.errors_empty = []
```

Comment:
This `errors_empty` we can also eliminate? At least I don't see any further usage of it, besides appending a few lines down.